fido-device-onboard.spec

```diff
@@ -0,0 +1,133 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global forgeurl https://github.com/fedora-iot/fido-device-onboard-rs
+
+%forgemeta
+
+Name:           fido-device-onboard
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        An implementation of the FIDO Device Onboard Specification written in rust
+
+License:        BSD 3
+URL:            https://github.com/fedora-iot/fido-device-onboard-rs
+Source:         %{url}/archive/%{commit}/%{name}-rs-%{commit}.tar.gz
+%if 0%{?rhel} && !0%{?eln}
+Source1:        %{name}-rs-%{commit}-vendor.tar.gz
+%endif
+
+ExclusiveArch:  %{rust_arches}
+# RHBZ 1869980
+ExcludeArch:    s390x i686 %{power64}
+
+%if 0%{?rhel} && !0%{?eln}
+BuildRequires:  rust-toolset
+%else
+BuildRequires:  rust-packaging
+%endif
+BuildRequires: systemd
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n %{name}-rs-%{commit} -p1
+%if 0%{?rhel} && !0%{?eln}
+%cargo_prep -V 1
+%else
+%cargo_prep
+%endif
+
+%build
+%cargo_build
+
+%install
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-client-linuxapp
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-manufacturing-client
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-manufacturing-server
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-owner-onboarding-server
+install -D -m 0755 -t %{buildroot}%{_libexecdir}/fdo target/release/fdo-rendezvous-server
+install -D -m 0755 -t %{buildroot}%{_bindir}/fdo target/release/fdo-owner-tool
+install -D -m 0644 -t %{buildroot}%{_unitdir} examples/systemd/*
+install -D -m 0644 -t %{buildroot}%{_docdir}/fdo examples/config/*
+# 52fdo
+install -D -m 0755 -t %{buildroot}%{dracutlibdir}/modules.d/52fdo dracut/52fdo/module-setup.sh
```

Comment:
I think this should just be called 52fdo IE drop the dracut in the module name as it's implied by the location.

Reply:
will do perfect :+1:

Reply:
there's no `dracut` in the name :thinking:

Reply:
It doesn't read like that here (or in the downloaded spec), and it looks like it has a space. copy from my local spec is ````%{dracutlibdir}/modules.d/52fdo dracut/52fdo/````:

![dracut](https://user-images.githubusercontent.com/1917405/142179615-9e9d9cb8-05ca-480e-9972-587d1d83dae3.png)

Reply:
I don't know where and how you're getting that :joy: the module is called 52fdo and in my testing it's indeed fdo to pull it in in dracut

Reply:
aahhhh that's just the `install` syntax - the space is for install TARGET SRC so doesn't have anything to do with the name too